tests/installation/ipxe_install.pm

```diff
@@ -147,9 +157,29 @@ sub enter_o3_ipxe_boot_entry {
     }
 }
 
-sub set_bootscript_cmdline_extra {
+my $cmdline_extra = " ";
 
-    my $cmdline_extra = " ";
+sub set_bootscript_agama_extra {
+    if (get_var('AGAMA_AUTO')) {
+        my $agama_auto = data_url(get_var('AGAMA_AUTO'));
+        $agama_auto =~ s/^\s+|\s+$//g;
+        $cmdline_extra .= "agama.auto=$agama_auto ";
+    }
+    # Agama Installation repository URL
+    # By default Agama installs the packages from the repositories specified in the product configuration.
+    # From now Agama supports using the agama.install_url boot parameter for overriding the default installation repositories.
+    if (my $agama_install_url = get_var('AGAMA_INSTALL_URL')) {
+        $agama_install_url =~ s/^\s+|\s+$//g;
+        $cmdline_extra .= "agama.install_url=$agama_install_url ";
+    }
+    if (is_ipmi) {
+        my $sol_console = is_aarch64 ? get_var('IPXE_CONSOLE', 'ttyAMA0') : get_var('IPXE_CONSOLE', 'ttyS1');
+        $cmdline_extra .= "console=$sol_console,115200 linuxrc.log=/dev/$sol_console linuxrc.core=/dev/$sol_console linuxrc.debug=4,trace ";
```

Comment:
`IPXE_CONSOLE` is in format `IPXE_CONSOLE=ttyAMA0,115200`.  It means it will not work.
Also logic with default values if confusing, Use `get_required_var`, every bare metal worker should have it, and split it to get correct tty.